themes/custom/droopler_theme/scss/components/_d_p_subscribe.scss

```diff
@@ -22,15 +22,39 @@ $d-p-subscribe-line-margin-top: 1.45rem !default;
 // Other settings.
 $d-p-subscribe-background-opacity: 0.4 !default;
 
+.theme-default {
+  .subscribe_file {
+    background: $d-p-subscribe-background;
+    a {
+      color: $d-p-subscribe-color;
+    }
+
+    .field--name-field-d-main-title {
+      color: $d-p-subscribe-header-color;
+    }
+  }
+}
+
 // Override default values.
 .padding-top-default {
   .paragraph--type--d-p-subscribe-file {
-    padding-top: 0;
+    &.paragraph:not(.content-moved-inside) {
+      padding-top: 0;
+    }
+    .subscribe_file {
+      padding-top: $d-p-subscribe-padding-y;
+    }
   }
 }
+
 .padding-bottom-default {
   .paragraph--type--d-p-subscribe-file {
-    padding-bottom: 0;
+    &.paragraph:not(.content-moved-inside) {
+      padding-bottom: 0;
+    }
+    .subscribe_file {
+      padding-bottom: $d-p-subscribe-padding-y;
+    }
   }
 }
```

Comment:
po prostu: 
nie trzeba tu uwzgledniac `.paragraph:not(.content-moved-inside)` bo tez w twigach tego paragrafu nie ma takich zmian
![image](https://user-images.githubusercontent.com/37533838/87315702-036a9580-c525-11ea-885c-fad48bde861c.png)